server/services/database/applicantSkills.js

```diff
@@ -45,4 +45,37 @@ const getSkillsForApplicantProfile = id => {
     );
   });
 };
-module.exports = { newApplicantSkills, getSkillsForApplicantProfile };
+const getSkillsForEditApplicantProfile = id => {
```

Comment:
This is same as getSkillsForApplicantProfile so you can getSkillsForApplicantProfile instead try to avoid duplicating things.

Reply:
i need the skills id the other functions get the and need to get it by the applicant profile id not the user id